pilot/prompts/components/steps_list.prompt

```diff
@@ -0,0 +1,24 @@
+{% if task_steps and step_index -%}
+Task is split into multiple steps. Previously generated steps can be one of the following:
```

Comment:
This sounds like some new steps will be generated which are different? Suggest change like:

The current task has been split into multiple steps, and each step is one of the following:

Reply:
updated